docs/user-guide/how_to_use_mpi_plugin.md

```diff
@@ -0,0 +1,78 @@
+# MPI Plugin User Guide
+
+## Introduction
+
+**MPI plugin** is designed to optimize the user experience when running MPI jobs, it not only allows users to write less yaml, but also ensures the normal operation of MPI jobs.
+
+## How the MPI Plugin Works
+
+The MPI plugin will do three things:
+
+* Add the `dependsOn` field to the master task, so that the worker starts and then starts the master
```

Comment:
As you know, MPI always use the gang, Is the `dependsOn` is allowed to work with gang together so far?

Reply:
Currently we do not support the coexistence of `dependsOn` and gang,but we have a preliminary draft from @HecarimV .
Could you please help to review it, if this method is ok, we will merge this code into the community.
@william-wang @Thor-wl @shinytang6 
https://github.com/HecarimV/volcano/pull/3

Reply:
In addition, it is just a brief review to see if this coexistence method is feasible. If it is feasible, we will create a PR in volcano and review it in detail in the PR of volcano

Reply:
Since coexistence with gang is not supported at present, I think maybe we can remove adding `dependsOn ` field and add it later once supported. The removal of this field will not affect the work of the MPI plugin.

Reply:
> Since coexistence with gang is not supported at present, I think maybe we can remove adding `dependsOn ` field and add it later once supported. The removal of this field will not affect the work of the MPI plugin.

Do I directly delete or comment out the relevant code?

Reply:
l think it's ok to comment out the codes and add some TODO comment